doc/whatsnew/2.12.rst

```diff
@@ -3,11 +3,25 @@
 ***************************
 
 :Release: 2.12
-:Date: TBA
+:Date: 2021-11-24
 
 Summary -- Release highlights
 =============================
 
+In 2.12, we introduced a new option ``py-version`` that permits to analyse code for a python
+version that may differ from your current python interpreter. This does not affect all checkers but
+permits, for example, to check for python 3.5 code smells (using f-string's) while using pylint with python 3.6.
+The minimum version to run pylint is now 3.6.2, while the last working version for python 3.6.0
+and 3.6.1 was pylint 2.9.3.
+
+On top of fixing a lot of false positives and bugs, we also added new default checks, like
+``use-implicit-booleaness-not-comparison``, ``overridden-final-method``, and ``useless-with-lock``.
+There's also better check for TOML configurations.
```

Comment:
```suggestion
There's also better check for TOML configurations.

Lastly, in addition to the information we already had about start line and start column,
we introduced new information about the end line and end column of messages. This
will permit to have more precise visual clue in IDE like in pylint for vs-code. The default
will stay the same to not break compatibility but it can be retrieved by adding ``end_line``
and ``end_column`` to the ``--msg-template`` option. For better result stick to python 3.8+.
```
@DanielNoord